### ISSUE TITLE
DeprecationWarning: Tapable.plugin is deprecated. Use new API on `.hooks` instead

### ISSUE THREAD

**Issue by alexgorbatchev** [mzgoddard/hard-source-webpack-plugin#299]
```
(node:50821) DeprecationWarning: Tapable.plugin is deprecated. Use new API on `.hooks` instead
    at HardSourceWebpackPlugin.apply (/Users/.../node_modules/hard-source-webpack-plugin/index.js:443:12)
    at webpack (/Users/.../node_modules/webpack/lib/webpack.js:37:12)
    at startDevServer (/Users/.../node_modules/webpack-dev-server/bin/webpack-dev-server.js:379:16)
```

Environment

```
node 9.3.0
"hard-source-webpack-plugin": "^0.6.4",
"webpack": "^4.2.0",
```

**Comment by kaloudis** [comment]
Seeing the same here - looks like a Webpack 4 compatibility issue

**Comment by alexgorbatchev** [comment]
It is webpack 4 relates. It's still working, I suspect this will break with webpack 5.

**Comment by zettamax** [comment]
Thanks all!

**Comment by flyyang** [comment]
Have time to fixe this?

**Comment by mzgoddard** [comment]
Fixed in `0.7.0-alpha.0`
